README.md

```diff
@@ -7,9 +7,8 @@ btcd
 btcd is an alternative full node bitcoin implementation written in Go (golang).
 
 This project is currently under active development and is in a Beta state.  It
-is extremely stable and has been in production use for over 6 months as of May
-2014, however there are still a couple of major features we want to add before
-we come out of beta.
+is extremely stable and has been in production use for just under 2 and a half
+years as of March 2016.
```

Comment:
Maybe say "in production use since ..." instead of updating this line each time?